.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+<!--
+ * Copyright (c) 2022,2023 Contributors to the Catena-X Association
```

Comment:
wrong header, applies to all files

Reply:
Yes, if I adapt the PR right now (or the content) which was already approved by EF, I guess I would have to ask again for approval.

We have to adapt Copyrights, legal documents, Workflows anyway. This was planned on Monday directly on the repository after initial merge.

Reply:
It was wrong to open the IP issue with these headers in. 
The approval is also wrong in my opinion. Maybe the automated check missed it, but these headers definitely need to be reworked

Reply:
We will adapt it and ask for a new review.